types/node/events.d.ts

```diff
@@ -130,10 +135,15 @@ declare module "events" {
      * It supports the following option:
      * @since v0.1.26
      */
-    class EventEmitter<T extends EventMap<T> = DefaultEventMap> {
+    class EventEmitter<Events extends EventMap<Events> = {}> {
```

Comment:
Is this declaration backwards compatible for those who had previously used a tuple here? I'm noticing that `DefaultEventMap` is `[never]`, but the new default is `{}`.

Reply:
a tuple was never acceptable, this default event map was a hack

Reply:
I mainly ask because I went scouring over GitHub to find people who had parameterized `EventEmitter` and found a couple of cases where they were tuples...

Reply:
If we're using the same search terms, those look to be namesakes from other APIs (Angular in particular)